base/doc/ltnews35.tex

```diff
@@ -183,6 +183,9 @@ \section{Document metadata interface}
 already existing keys are in \file{l3meta.pdf} and \file{documentmetadata-support.pdf}
 and also in the documentation of the \pkg{pdfmanagement-testphase} package.
 
+Package and class authors can test if a user has used \cs{DocumentMetadata}
+and so loaded the pdfmanagement package with \cs{IfPDFManagementActiveTF}. 
+
```

Comment:
My initial reaction would be that this is the wrong interface -- convince me otherwise :-) I think it makes more sense going forward to have a test that  would be called \IfDocumentMetadataTF because that is the deciding factor from an interface perspective

Reply:
Well that command is not new code. It was only undocumented and so I added a sentence. But yes, I think in view of the development the focus has switched a bit and \IfDocumentMetadataTF is a better name.